docs/src/main/paradox/getting-started/testing.md

```diff
@@ -9,14 +9,14 @@ artifact=akka-projection-testkit_$scala.binary.version$
 version=$project.version$
 }
 
-Import the @apidoc[ProjectionTestKit] and other utilities into a new [ScalaTest](https://www.scalatest.org/) test spec.
+Import the @apidoc[akka.projection.testkit.(javadsl|scaladsl).ProjectionTestKit] and other utilities into a new [ScalaTest](https://www.scalatest.org/) test spec.
```

Comment:
It's supposed to handle this case with finding the javadsl/scaladsl apis with the same name, surprising that it didn't. Are there more classes named ProjectionTestKit in addition to these two?

Reply:
I learned that today. I saw it on this same page: `@apidoc[akka.actor.testkit.typed.(javadsl|scaladsl).ActorTestKit]`. See next diff.